css/design_system/utils/colors.scss

```
@@ -1,54 +1,60 @@
 $colors: 
-  "--s-color-black", 
-  "--s-color-white", 
-  "--s-color-cyan-400",
-  "--s-color-cyan-500",
-  "--s-color-cyan-600",
-  "--s-color-cyan-700",
-  "--s-color-gray-100",
-  "--s-color-gray-200",
-  "--s-color-gray-300",
-  "--s-color-gray-400",
-  "--s-color-gray-500",
-  "--s-color-gray-600",
-  "--s-color-gray-700",
-  "--s-color-gray-800",
-  "--s-color-red-500",
-  "--s-color-red-600",
-  "--s-color-red-700",
-  "--s-color-yellow-500",
-  "--s-color-yellow-600",
-  "--s-color-yellow-700";
+  "--s-color-black" false, 
+  "--s-color-white" true, 
+  "--s-color-cyan-400" true,
+  "--s-color-cyan-500" true,
+  "--s-color-cyan-600" true,
+  "--s-color-cyan-700" false,
+  "--s-color-gray-100" true,
+  "--s-color-gray-200" true,
+  "--s-color-gray-300" true,
+  "--s-color-gray-400" true,
+  "--s-color-gray-500" true,
+  "--s-color-gray-600" false,
+  "--s-color-gray-700" false,
+  "--s-color-gray-800" false,
+  "--s-color-red-500" true,
+  "--s-color-red-600" false,
+  "--s-color-red-700" false,
+  "--s-color-yellow-500" true,
+  "--s-color-yellow-600" true,
+  "--s-color-yellow-700" true;
 
-@mixin docstring($class, $util_type, $color_type, $text) {
+@mixin docstring($class, $util_type, $color_type, $text, $example_class) {
```

Suggestion for slight change to variable naming:
```suggestion
@mixin docstring($class, $util_type, $color_type, $text, $example_classes) {
```